static/js/redux/ui/Calendar.tsx

```diff
@@ -55,8 +58,27 @@ const Row = (props: RowProps) => {
   );
 };
 
+export const ShowWeekendsButton = () => {
```

Comment:
export it so it can be used in DayCalendar.tsx too

Reply:
The mobile Calendar can't "show weekends" since it's day-by-day. We can remove it entirely

Reply:
I'll comb it as part of my review, so don't worry about it